logpyle/runalyzer.py

```diff
@@ -353,11 +364,15 @@ def execute_magic(self, cmdline):
             from pylab import title
             title(args)
         elif cmd == "plot":
+            import re
+            xlabel, ylabel = re.split(", +|,| +", args)[1:3]
```

Comment:
This won't work properly half the time. Instead, get the column labels from the DBMS.

Reply:
note to self: https://stackoverflow.com/questions/947215/how-to-get-a-list-of-column-names-on-sqlite3-database

Reply:
Done (but required some SQL gymnastics since `mange_sql` destroys the column names).